security/pkg/nodeagent/caClient/client_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ca
+
+import (
+	"context"
+	"net"
+	"reflect"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+
+	capb "istio.io/istio/security/proto/ca/v1alpha1"
+)
+
+const (
+	mockServerAddress = "localhost:0"
```

Comment:
use port other than 0?

Reply:
`0` makes it allocated from available ports.

Reply:
ok